data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc

```diff
@@ -7,7 +7,7 @@ GENRE:Fantasy
 BOOKTYPE:Setting Supplement
 SETTING:Fantasy
 STATUS:ALPHA
-PRECAMPAIGN:1,INCLUDES=Player's Guide to Faerun
+PRECAMPAIGN:2,INCLUDES=Player's Guide to Faerun,Arms and Equipment Guide
```

Comment:
My version means AE is optional, so is prefered.
However i was only looking at dupe equipment, the other fixes may be needed.

Cheers
Dave
On 12 Mar 2016 04:22, "Bahamut" <notifications@github.com> wrote:

> In
> data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc
> <https://github.com/BahamutDragon/pcgen/pull/420#discussion_r55915853>:
>
> > @@ -7,7 +7,7 @@ GENRE:Fantasy
> >  BOOKTYPE:Setting Supplement
> >  SETTING:Fantasy
> >  STATUS:ALPHA
> > -PRECAMPAIGN:1,INCLUDES=Player's Guide to Faerun
> > +PRECAMPAIGN:2,INCLUDES=Player's Guide to Faerun,Arms and Equipment Guide
>
> How does this reconcile with the changes @FerretDave
> <https://github.com/FerretDave> just made?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BahamutDragon/pcgen/pull/420/files#r55915853>.
>